x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/v2/RollupActionSingleNodeTests.java

```diff
@@ -214,6 +215,7 @@ public void setup() {
                 "type=double"
             )
             .get();
+        assertTrue("Source index was not created", r.isAcknowledged());
```

Comment:
can it replace by assertAcked?

Reply:
Added `assertAcked()` everywhere possible in the test here: [a5b09f9](https://github.com/elastic/elasticsearch/pull/89025/commits/a5b09f9afe573ff03386bf3e84d6f2e81795095e)